catalog/resource_share.go

```diff
@@ -175,6 +175,7 @@ func (beforeSi ShareInfo) Diff(afterSi ShareInfo) []ShareDataChange {
 
 func ResourceShare() common.Resource {
 	shareSchema := common.StructToSchema(ShareInfo{}, func(m map[string]*schema.Schema) map[string]*schema.Schema {
+		m["name"].DiffSuppressFunc = common.EqualFoldDiffSuppress
```

Comment:
Should we have it for names of objects in the share as well?

Reply:
@alexott, are the share object names case insensitive? I didn't find the information here: https://docs.databricks.com/en/data-sharing/create-share.html

Reply:
I don't remember, but they could be if we refer to specific objects - tables, volumes and models.
@nkvuong do you remember?